standards/3-commercial/com/pricing/linear-model.md

```diff
@@ -0,0 +1,103 @@
+# Linear pricing model
+
+This document describes how to compute the price of a Golem service using
+a linear pricing model.
+
+## Properties
+
+The `Linear` model uses the following properties:
+
+- [`golem.com.pricing.model`](model.md#golemcompricingmodel--string) - Setting to `linear` indicates that the linear pricing model is used.
+- [`golem.com.pricing.model.linear.coeffs`](model.md#golemcompricingmodellinearcoeffs--listnumber) -
+   An array of price coefficients for the linear pricing function.
+
+```
+coeffs = [coeff_1, ..., coeff_N, coeff_fixed]
+```
+
+- [`golem.com.usage.vector`](../usage.md#golemcomusagevector--liststring) - Array of usage counters for resources consumed by a given activity.
+
+```
+counters = [counter_1, ..., counter_N] 
+```
+
+In general, the linear pricing model assumes that the total price for an activity is a sum of products of coefficients of a usage array and a coeffs array:
+
+```
+price = usage_1 x coeffs_1 + ... + usage_N x coeffs_N + 1 x coeff_fixed.
+```
+
+where `n` denotes an index of a particular measure agreed between the provider and the requestor to be used for this agreement and `usage` is the actual resource usage per counter.
+
+## Usage Counters
+
+Usage `counters` are measures of resource consumption during the execution of the task or service. Each ExeUnit implements a specific subset of usage counters and reports their values to the Yagna daemon. It is the Provider's choice to decide which counters he uses to measure the resource consumption and finally calculate the price. Provider provides this information in the offer as the `golem.usage.vector`.
+
+A `golem.com.usage.vector` may be set as i.e. `["golem.usage.cpu_sec",
+   "golem.usage.duration_sec"]`
+ which indicates that the price will be a function of both the time of CPU and the environment run. The actual price value will depend on the actual coeffs values.
+
+The order of the counters in the `golem.com.usage.vector` is important. The same order is used for the price coefficient array, so the coefficient on position n (`coeff_n`) corresponds to the usage counter on position n (`counter_n`). The provider will report the consumption also using this order.
+
+Usage counters are reported from the moment ExeUnit process starts.
+
+## Price Coefficients
+
+Price coefficients `golem.com.pricing.model.linear.coeffs` on the other side, contain prices of specific counters declared in the `usage` vector. Notice that the `coeffs` array has one element more than the `counters` vector, which always represents a fixed cost for starting an Activity. These values are decided by the provider and are subject to the agreement.
```

Comment:
```suggestion
Price coefficients `golem.com.pricing.model.linear.coeffs` on the other side, contain prices of specific counters declared in the `usage` vector. Notice that the `coeffs` array has one element more than the `counters` vector, which always represents a fixed cost for starting an activity. These values are decided by the provider and are subject to the agreement.
```

Reply:
In most of our docs we write yagna artifacts with capital letter